chef/data_bags/crowbar/migrate/network/101_add_ironic_network.rb

```diff
@@ -0,0 +1,20 @@
+def upgrade(ta, td, a, d)
+  unless a["networks"].key? "ironic"
+    a["networks"]["ironic"] = ta["networks"]["ironic"]
+  end
+
+  # copy intf1 as intf3
+  a["conduit_map"].each do |cm|
+    unless cm["conduit_list"].key? "intf3"
+      cm["conduit_list"]["intf3"] = cm["conduit_list"]["intf1"]
```

Comment:
I am asking this, because when you put both ```admin``` and ```ironic``` untagged onto the same physical interface(s) and both networks provide dhcp/pxe for the network that is not goint to work out.

One solution to this might be to make ```ironic``` a tagged VLAN on the compute nodes and reconfigure the switch so that that VLAN will be the native VLAN on the ports from which the bare-metal instances boot.
The other option is to require ```ironic``` to be on a separate physical network, I guess.